packages/data-models/appConfig.ts

```diff
@@ -192,6 +192,16 @@ const APP_UPDATES = {
   app_update_downloaded_field: "_app_update_downloaded",
 };
 
+type ISupabaseConfig = {
```

Comment:
nit(blocking)
As mentioned in other comment, better if we can move this to main deployment model instead